source/mock/data.json

```
@@ -1,4 +1,63 @@
 {
-  "foo": "bar",
-  "lorem": "ipsum"
+  "navLinks": [
+    {
+      "href": "index.html",
+      "text": "О сервисе"
+    },
+    {
+      "href": "form.html",
+      "text": "Направления"
+    },
+    {
+      "href": "catalog.html",
+      "text": "Попутчики"
+    }
+  ],
+
+  "userInfo": [
+    {
+      "userSrc": "firsova-mobile",
+      "userAlt": "Фото Тани Фирсовой",
+      "name": "Таня",
+      "surname": "Фирсова",
+      "like": "1,5 М",
+      "hashtag": "['#ЗОЖ', '#ПП', '#Фитнес', '#пляж', '#авокадо', '#смузи']",
+      "countrySrc": "['Sri-Lanka-small', 'Thailand-small', 'Seychelles-small']",
+      "countryAlt": "['Шри-ланка', 'Таиланд', 'Сейшелы']",
+      "level": "99"
```

Лучше каждому юзеру добавить хобби с булевым занчением
```suggestion
      "level": "99"
       "hobbies": {"plane": true, "bus": false, "bicycle": false, "run": false}
```

И в этот же файл отдельно написать объект ассоции 
```json
"HOBBIES_RUS": {"plane": "авиаперевозки", "bus": "автобусные путешествия", "bicycle": "кататься на велосипеде", "run": "пешие прогулки"}
```